.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "4.1"
+  - "6.1"
```

Comment:
we should just go with "6" as that will always raise it to the latest of Node.js instead of being "stuck" at 6.1.